app/scripts/views/settings/account_recovery/confirm_revoke.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Cocktail = require('cocktail');
+const FormView = require('../../form');
+const PasswordMixin = require('../../mixins/password-mixin');
+const ModalSettingsPanelMixin = require('../../mixins/modal-settings-panel-mixin');
+const Template = require('templates/settings/account_recovery/confirm_revoke.mustache');
+
+const View = FormView.extend({
+  template: Template,
+  className: 'account-recovery-confirm-revoke',
+  viewName: 'settings.account-recovery.confirm-revoke',
+
+  events: {
+    'click .cancel-link': '_returnToAccountRecovery'
+  },
+
+  submit() {
+    const account = this.getSignedInAccount();
+    return account.deleteRecoveryKey()
```

Comment:
This does not display network errors. I tried to delete the account recovery key on an account where it wasn't set, I see a 500 error in the console but no error in the UI.

Reply:
I wasn't able to replicate this locally with those steps. The delete request 500?

Reply:
This might be related to https://github.com/mozilla/fxa-content-server/pull/6392#discussion_r206270298, this route was also updated to be singular.

Reply:
You are right, the shrinkwrap file for the auth-server needs to be updated with the newest db-server.